books.js

```diff
@@ -0,0 +1,77 @@
+/* eslint-disable no-unused-vars */
+/* eslint-disable camelcase */
+/* eslint-disable max-classes-per-file */
```

Comment:
- [ ] It is not allowed to disable any linter rule because you are creating a room for making syntax errors that can introduce bugs in the codebase and these can be hard to debug than it is to find how to correct those errors flagged by linters. Please remove these statements and then find how you can correct every error flagged by `eslint`  this will help you write good codes 👍